patches/0036-Disable-retry-for-KMS-access-denied-in-lambda.patch

```diff
@@ -44,25 +45,34 @@ index 0000000000..54f6aac15a
 +	})
 +	return lambda_sdkv2.NewFromConfig(cfg, func(o *lambda_sdkv2.Options) {
 +		if endpoint := config[names.AttrEndpoint].(string); endpoint != "" {
++			tflog.Debug(ctx, "setting endpoint", map[string]any{
```

Comment:
This change comes from this upstream PR: https://github.com/hashicorp/terraform-provider-aws/pull/34233/files#diff-9c93e6008819246a3298682bf745457767b75aad2ce53cd024233e30253b4c9cR165

Custom endpoints can now override the `use_fips_endpoint` flag for individual services